### DISCUSSION TITLE
Reward high health value

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#4395] [General]
- increase stamina regen
- higher max stamina pool
- reduce pain penalty

**Comment by Ali-Anomma** [comment]
make the healths role in bacterial/fungal infection rolls much bigger. Max health players can survive the imfection without antibiotics.

**Reply by chaosvolt** [reply]
That's already a thing, just buried in the code. Dunno if it even has enough effect to be worth it though.

**Reply by Ali-Anomma** [reply]
yeah but it is still tied to luck, guaranteeing a max health player will survive the infection without antibiotics would be a good addition.

**Reply by Ali-Anomma** [reply]
Someone also said it can increase the chance of good mutations but would it be balanced for mutations?

it can also reduce the chance of things like bleeding/infection happening

it can provide a small boost to bodily skills such as swimming, melee or ranged
